packages/client/src/pages/welcome.entrance.a.vue

```diff
@@ -28,6 +28,7 @@
 				<div class="action">
 					<MkButton inline rounded gradate data-cy-signup style="margin-right: 12px;" @click="signup()">{{ i18n.ts.signup }}</MkButton>
 					<MkButton inline rounded data-cy-signin @click="signin()">{{ i18n.ts.login }}</MkButton>
+					<MkButton inline rounded style="margin-left: 12px;" onclick="window.location.href='/featured'">{{ i18n.ts.guest }}</MkButton>
```

Comment:
If it is a link you can use `link` and `to` as seen in:
https://github.com/misskey-dev/misskey/blob/01d5a97a4fab5f311f977b897e779a1393c1cd05/packages/client/src/pages/my-antennas/index.vue#L5

```suggestion
					<MkButton :link="true" to="/featured" inline rounded style="margin-left: 12px;">{{ i18n.ts.guest }}</MkButton>
```